libs/sketchtypes/src/lib/sketchtypes.ts

```diff
@@ -483,4 +489,5 @@ export interface SketchMSData {
   document: SketchMSDocument;
   user: SketchMSUser;
   meta: SketchMSMeta;
+  fileData: SketchFile;
```

Comment:
I'm not sure if this change is the correct one. The types we have defined here are types related to a Sketch Design file. Basically, this is our map of data of the ingested Sketch file.

I think we should remove it here and add this to the `UISettings` interface in our `ui.state.ts` as I believe this fits better in the generic parts of ingesting design files. 

Do you agree with this and what is your oppinion @manekinekko ? 

I'm happy to assist you with this btw, so DM me if you would like a chat.

Reply:
Yes you're right. I added this quickly and I didn't think about the best place to put it.
Firstly I wanted to put it inside 'meta' object, but I realized it wasn't the right place either. I'll fix it.
Thank you for you help :-)

Reply:
Great work so far ☺️

I'd also suggest the following changes:
- change name from `SketchFile` to something generic like `DesignFile`
- change typing to:
```typescript
interface DesignFile {
  name: string;
  icon: string;
}
```
- And as @Jefiozie  suggested, move the file data description to the `UISettings`